torchvision/prototype/models/detection/keypoint_rcnn.py

```diff
@@ -24,6 +24,9 @@
 
 
 _COMMON_META = {
+    "task": "image_object_detection",
+    "architecture": "FasterRCNN",
+    "publication_year": 2015,
```

Comment:
Same comments as for MaskRCNN -- KeypointRCNN was introduced in the Mask R-CNN paper. In this case you could potentially use the same `architecture` though, although I could also see it being its own architecture